Justfile

```diff
@@ -175,11 +175,32 @@ debug *args:
   dlv_pid=$!
   wait "$dlv_pid"
 
+# grpcPort is used for observing the system via otel.
+# otel is short for OpenTelemetry.
+grpcPort := `cat docker-compose.yml | grep "OTLP gRPC" | sed 's/:.*//' | sed -r 's/ +- //'`
```

Comment:
Since this is a top-level var, maybe we should name it `otelGrpcPort`? Then, you might be able to remove this comment `grpcPort is used for observing the system via otel`

Reply:
ah great call!